### ISSUE TITLE
Color contrast: placeholder text

### ISSUE THREAD

**Issue by aviannamiller** [omeka/omeka-s#2372]
Within text inputs that accompany `.color-picker` selectors, placeholder text fails color contrast requirements (at 2.2:1). [Text at this size should have a contrast of at least 4.5:1, even if for placeholder purposes](https://accessibleweb.com/question-answer/does-placeholder-text-need-to-meet-color-contrast-requirements/). This is applicable to the theme settings of Papers, The Daily, Default, Cozy, and Thanks, Roy.

There may very well be other areas of the interface with a similar issue.

**Comment by aviannamiller** [comment]
The update is made on Firefox, but it looks like the same changes need to be applied to `::-webkit-input-placeholder` to target other browsers.
